examples/misconf/custom-policy/terraform-plan/README.md

```diff
@@ -1,5 +0,0 @@
-# Terraform Plan
-
-```
-$ trivy conf --policy ./policy --namespaces user ./configs
-```
```

Comment:
@knqyf263 - was this moved somewhere else? I am unable to find it anywhere. As a result it has broken most of the links on this page: https://aquasecurity.github.io/trivy/v0.41/docs/scanner/misconfiguration/custom/examples/